BUILD

```diff
@@ -1,13 +1,37 @@
 package(default_visibility = ["//visibility:public"])
 
-ALL_MODULES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ASMCLI_MODULES = glob(["asmcli/**"])
+NODE_MODULES = glob(["node_modules/**"])
+ASMCLI_SOURCES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ALL_UNIT_TESTS = glob(["asmcli/tests/*.bats"])
 MERGE_OUT = ["asmcli"]
 
 genrule(
     name = "merge",
-    srcs = ALL_MODULES,
+    srcs = ASMCLI_SOURCES,
     outs = MERGE_OUT,
     cmd = "$(location scripts/release-asm/merge) $(OUTS) $(SRCS)",
     tools = ["scripts/release-asm/merge"],
     executable = True,
 )
+
+sh_library(
+    name = "asmcli_modules",
+    data = ASMCLI_MODULES,
+)
+
+sh_library(
+    name = "node_modules",
+    data = NODE_MODULES,
+)
+
+sh_test(
+    name = "unit_test",
```

Comment:
I think these are traditionally just called "test", but if you saw another example somewhere ignore me.

Reply:
Done